stubs/auth/routes/web.php

```diff
@@ -1,5 +1,13 @@
 <?php
 
+use App\Http\Controllers\Auth\EmailVerificationController;
```

Comment:
Maybe a personal preference but I'm not a fan of having a use statement for every single route, it doubles the size of the file in a not-so-nice way.

How about just importing the partial namespaces:

```php
use App\Http\Controllers;
use App\Http\Livewire;
```

Then, later on, we can reference them inline without needing the whole thing:

```php
Route::get('login', Livewire\Auth\Login::class);
```

Reply:
Yeah, I was going to do this but probably best to stay with the current way to please most people. I don't mind changing it though if you think it's better.